fuzzbucket/app.py

```diff
@@ -386,6 +386,44 @@ def create_box():
     )
 
 
+@app.route("/box/<string:instance_id>", methods=["PUT"])
+def update_box(instance_id):
+    if not is_fully_authd():
+        return auth_403_github()
+    log.debug(
+        f"handling update_box for user={request.remote_user!r} "
+        + f"instance_id={instance_id!r}"
+    )
+    if instance_id not in [
+        b.instance_id
+        for b in list_user_boxes(
+            get_ec2_client(), request.remote_user, os.getenv("CF_VPC")
+        )
+    ]:
+        return jsonify(error="no touching"), 403
+
+    instance_tags = []
+    for key, value in request.json.get("instance_tags", {}).items():
+        tag_spec = dict(Key=str(key), Value=str(value))
+
+        log.debug(f"adding tags from request json 'instance_tags' spec={tag_spec!r}")
+
+        instance_tags.append(tag_spec)
```

Comment:
If an existing tag key is re-added here, what is the behavior? Can a tag be removed (e.g. by a null value)?

Reply:
I was assuming at this point that tag removal isn't handled 🤷🏼 🤔

Reply:
:+1: